_episodes/00-quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Quickstart Guide"
+teaching: 5
+exercises: 10
+questions:
+
+- "How do I activate the ESMValTool environment?"
+- "How do I configure the ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Activate the ESMValTool environment"
+- "Configure the ESMValTool"
+- "Successfully run a recipe"
+
+keypoints:
+- "See the installation guide for activating the ESMValtool environment."
+- "The config file has to be set up, uncommenting the environment you use."
+- "The command for running a recipe is `esmvaltool run`."
+---
+
```

Comment:
Plaese add an introductory paragraph about what the learner will learn in this lesson, and who are the target audience.